index.html

```diff
@@ -163,13 +170,39 @@ <h2>
       </dd>
       <dt>
         <dfn data-dfn-for="digital credential" data-local-lt=
-        "response">Presentation response</dfn>
+        "presentation response">Presentation response</dfn>
       </dt>
       <dd>
         A format that a [=holder|holder's=] software, such as a digital wallet,
         uses, via an [=digital credential/exchange protocol=], to respond to a
         [=digital credential/request data=] by a [=verifier=].
       </dd>
+      <dt>
+        <dfn data-dfn-for="digital credential" data-local-lt=
+        "issuance request">Issuance request</dfn>
+      </dt>
+      <dd>
+        An issuance request is an offer to issue a [=digital credential=]
+        composed of an [=digital credential/credential offer=] and a [=digital
+        credential/exchange protocol=].
+      </dd>
+      <dt>
+        <dfn data-dfn-for="digital credential">credential offer</dfn>
+      </dt>
+      <dd>
+        A format that [=issuer=] software or a [=user agent=] uses, via an
+        [=digital credential/exchange protocol=], to offer issuing a [=digital
+        credential=] from an [=issuer=].
+      </dd>
+      <dt>
+        <dfn data-dfn-for="digital credential" data-local-lt=
+        "issuance response">Issuance response</dfn>
+      </dt>
+      <dd>
+        A format that a [=holder|holder's=] software, such as a digital wallet,
+        uses, via an [=digital credential/exchange protocol=], to respond to an
+        [=digital credential/credential offer=] by an [=issuer=].
```

Comment:
Tweaking @marcoscaceres' [suggestion](https://github.com/WICG/digital-credentials/pull/204/files#r1976434412)
```suggestion
        A format that [=holder=] software, such as a digital wallet,
        uses, via an [=digital credential/exchange protocol=], to respond to a
        [=digital credential/credential offer=] by an [=issuer=].
```